web/src/routes/(user)/albums/[albumId=id]/[[photos=photos]]/[[assetId=id]]/+page.ts

```diff
@@ -6,7 +6,7 @@ import type { PageLoad } from './$types';
 export const load = (async ({ params }) => {
   await authenticate();
   const [album, asset] = await Promise.all([
-    getAlbumInfo({ id: params.albumId, withoutAssets: true }),
+    getAlbumInfo({ id: params.albumId, withoutAssets: false }),
```

Comment:
This is necessary as otherwise the albums with assets get deleted when clicking the back button.

Not sure if this is a performance problem/how expensive the query is in the backend.

Reply:
It is and it is quite expensive. Album info should have the asset count in it though.

Reply:
This works perfectly, no idea why I missed the property, sorry and thx.